docs/ui-style-guide.md

```diff
@@ -0,0 +1,66 @@
+# Machinery UI Style Guide
+
+Machinery has two user facing interfaces, the command line and the graphical user interface in the form of an HTML view.  
+All additions to Machinery should follow the same or at least a similar style to allow a consistent user experience.
+
+## Command Line Interface
+
+### Output
+
+The Markdown syntax is used to format the data when applicable, for example in headlines, listings and so on:
+
+```
+# Services [example] (2016-07-05 17:07:38)
+
+  * after-local.service: static
+  * auditd.service: enabled
+  * auth-rpcgss-module.service: static
+  * autovt@.service: disabled
+
+```
+
+Additional attributes of a list are bracketed, if it still fits one line of ~80 characters:
+```
+* bin (bin, uid: 1, gid: 1, shell: /bin/bash)
+```
+```
+*util-linux-2.25-12.2.x86_64 (openSUSE)
+```
+
+If not, the data is formatted as a list entry where the following lines are indented on the same level as the text of the first line:
```

Comment:
the relation to the condition of fitting one line is not very clear. Better repeat that and write 'If it exceeds 80 characters' instead of 'If not'.

'the following lines' can be interpreted as lines following this sentence.